### PR TITLE
Updating Main Readme to refer to new Directory Structure changes

### PR DESCRIPTION
**Pull request by salameer**
I've added some of the information that was sent out to the service teams

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X ] The title of the PR is clear and informative.
- [X ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X ] If applicable, the PR references the bug/issue that it fixes.
- [X ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X ] My spec meets the review criteria:
  - [ X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ X] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,11 +56,59 @@ The structure should appear like so:
...
+- Microsoft.ContainerService/2017-01-31/containerService.json
+
+Please take a look at this [readme.md](https://github.com/Azure/azure-rest-api-specs/tree/reorg/specification/compute/resource-manager) to understand how the complete structure looks like.
+
+
+## Specs Format
 Currently, the specifications are expected to be in Swagger JSON format
 
 ## Next steps
 The next step in the process after a spec is completed is to generate SDKs and API reference documentation. Go to the [Azure Developer Experience guide](https://github.com/Azure/adx-documentation-pr) for more information.
```
Comment (kirthik) [inline README.md]: adx-documentation-pr is a private repository. Do we want to link it from here? If yes, then we should be explicit that repo access is available only for Azure members.

File: README.md [hunk 2]
```diff
@@ -56,11 +56,59 @@ The structure should appear like so:
 ├── documentation
 └── readme.md
 ```
+## New Directory structure (effective July 10th 2017)
 
+Re-organization is being done to support upcoming profiles work and Data Plane swagger specs. The new structure will appear like below
```
Comment (kirthik) [inline README.md]: The directory structure does not show up correctly when you view it as .md. Text shows up in a single line. Could you please take a look?

File: README.md [hunk 3]
```diff
@@ -56,11 +56,59 @@ The structure should appear like so:
...
+|           +---2017-02-27-preview
+|           |   \---examples
+|           \---2017-04-24-preview
+|               \---examples
+└── readme.md
+ 
+### Changing the representation of composite swagger metadata
+
+Currently composite swagger metadata is captured in a .json file. The problem with the current representation is that it does not provide a way to track what versions of swagger specs were used in earlier releases. With the upcoming changes, we will be able to keep track of multiple versions of composition. The metadata will be captured in readme.md file under resource-manager folder. There won’t be a need for .json file. New structure in readme.md file looks like below. 
+Tag: 2017-03-30
```
Comment (kirthik) [inline README.md]: Tag name recommendation is changed to package-2017-03 so folks don't confuse this with API version. Could you please fix `Tag` here accordingly

File: README.md [hunk 4]
```diff
@@ -56,11 +56,59 @@ The structure should appear like so:
...
+These settings apply only when --tag=2017-03-30 is specified on the command line.
```
Comment (kirthik) [inline README.md]: Please fix tag name here too

File: README.md (hunk 1 above)
Comment (kirthik) [inline README.md]: It looks like we provided this info at the top of this page. It does not hurt to say it again but I will let you decide that.

File: README.md (hunk 2 above)
Comment (salameer) [inline README.md]: addressed that in the latest PR

File: README.md (hunk 3 above)
Comment (salameer) [inline README.md]: Sure Captain

**Pull request closed without merging**